portfolio-product/name.abuchen.portfolio.product

```diff
@@ -62,20 +67,27 @@
    </features>
 
    <configurations>
+      <plugin id="com.twelvemonkeys.imageio.bmp" autoStart="true" startLevel="4" />
       <plugin id="name.abuchen.portfolio.ui" autoStart="false" startLevel="4" />
-      <plugin id="com.twelvemonkeys.imageio.bmp" autoStart="true" startLevel="4"/>
-      <plugin id="org.eclipse.core.runtime" autoStart="true" startLevel="4" />
+      <plugin id="org.apache.aries.spifly.dynamic.bundle" autoStart="true" startLevel="1" />
       <plugin id="org.apache.felix.scr" autoStart="true" startLevel="2" />
+      <plugin id="org.eclipse.core.runtime" autoStart="true" startLevel="4" />
       <plugin id="org.eclipse.equinox.common" autoStart="true" startLevel="2" />
       <plugin id="org.eclipse.equinox.simpleconfigurator" autoStart="true" startLevel="1" />
-      <plugin id="org.apache.aries.spifly.dynamic.bundle" autoStart="true" startLevel="1" />
       <plugin id="org.eclipse.osgi" autoStart="true" startLevel="-1" />
       <property name="osgi.instance.area.default" value="$LOCALAPPDATA$/PortfolioPerformance/workspace" os="win32" />
       <property name="osgi.instance.area.default" value="@user.home/Library/Application Support/name.abuchen.portfolio.product/workspace" os="macosx" />
       <property name="osgi.instance.area.default" value="@user.home/.PortfolioPerformance/workspace" os="linux" />
-      <property name="osgi.requiredJavaVersion" value="17" />
+      <property name="osgi.requiredJavaVersion" value="21" />
       <property name="org.eclipse.update.reconcile" value="false" />
       <property name="eclipse.buildId" value="${unqualifiedVersion}.${buildQualifier}" />
    </configurations>
 
+   <preferencesInfo>
+      <targetfile overwrite="false"/>
+   </preferencesInfo>
```

Comment:
What does this property do?

Reply:
This was automatic added by Eclipse.
We can remove it ...

Reply:
I have no opinion yet. I do not know what this property does.

Reply:
I checked. We are not using additional plugin configuration. And even if we used it, we would want to not overwrite it. So I think it is safe to keep it there (and thereby keep the file up-to-date with the latest Eclipse framework).